setup.cfg

```
@@ -163,6 +164,7 @@ bundle_run =
     wheel
     pims
     numpy==1.19.3
+    napari-plugin-manager <0.2.0
```

```suggestion
    napari-plugin-manager >=0.1.0a0, <0.2.0
```
It looks like pip do not allow to install pre releases when using version constraints that do not contain proper suffix. It may ve connected with "0.2.0a0 < 0.2.0".